tests/lib/rules/new-cap.js

```diff
@@ -62,10 +62,12 @@ ruleTester.run("new-cap", rule, {
 
         { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptions: ["Bar"] }] },
         { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptions: ["Foo.Bar"] }] },
-        { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptionPattern: "^Foo\.." }] },
+
+        { code: "var x = Foo.Bar(42);", options: [{ capIsNewExceptionPattern: "^Foo\.." }] }, // eslint-disable-line no-useless-escape
```

Comment:
Pretty sure this should be `capIsNewExceptionPattern: "^Foo\\.."` (value is a string that we want interpreted as a regex, so we need to escape the backslash here).

This test is probably passing by accident because `\.` is generously interpreted as `.`, so the real pattern is `Foo..`, and `"Foo.B"` does match that pattern.